content/03.aim1.md

```diff
@@ -29,4 +29,34 @@ We will evaluate resulting models for their applicability to rheumatic disease a
 
 ### Results
 
-TBD.
+#### VAE test on simulated single cell datasets
+#### 1. Simulation data generation: 
+Simulated single cell data was generated by splatter [@doi:10.1186/s13059-017-1305-0]. 
+
+Parameters used:
+* nCells - The number of cells to simulate: 500 - 5000
+* nGenes - The number of genes to simulate: 20000 - 6000
+* nGroups - The number of celltypes: 5 - 15
+* outlier - probability of a gene that is an expression outlier:  0.1 - 0.5
+* default parameter is ncells = 600, nGenes = 20000, celltypes = 5, batchsize = 1
+
+Simulation statistics under different parameters:
+
+![](images/mean.gene.exp.bmp)
+
+Figure 1: Mean expression level of simulated genes
+
+#### 2. Visualization of simulated single cell data using VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE and PCA under different parameters
+
+  To see if different methods can recover cell types, we compared the 2D visualizations on simulated datasets. 2-layer VAE performs much better to differentiate different cell types when comparing with 1-layer VAE. The performance of VAE and t-SNE is similar and much better than PCA, but with the increase of outlier genes, 2-layer VAE is more resistant to noise.
+
+![](images/outlier.vis.png)
+
+Figure 2: 2 - layer VEA is more resistant to outlier. 2D visualization of simulated single cell data from different outlier parameters (0.1 - 0.5).
+
+#### 3. Performance evaluation of different simulation parameters
+Clustering performance was measured by normalized mutual information (NMI).
+
+![](images/nmi.png)
+
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters. 
```

Comment:
Just to make sure - this is comparing true labels against the labels called by the algorithm, right?

Reply:
Yes. This is comparing with labels called by k-means against the true labels